src/ui/templates/results/verticalresults.hbs

```diff
@@ -69,11 +69,11 @@
   {{/if}}
 {{/inline}}
 
-{{#*inline "viewMore"}}
+{{#*inline "viewAll"}}
   {{#if _config.isUniversal}}
-    <div class="yxt-Results-viewMore">
-      <a class="yxt-Results-viewMoreLink" href="{{ verticalURL }}">
-        <div class="yxt-Results-viewMoreLabel">{{_config.viewMoreLabel}}</div>
+    <div class="yxt-Results-viewAll">
+      <a class="yxt-Results-viewAllLink" href="{{ verticalURL }}">
+        <div class="yxt-Results-viewAllLabel">{{_config.viewMoreLabel}}</div>
```

Comment:
We still want to use {{_config.viewMoreLabel}} instead of the older {{_config.viewAllText}}. Setting only config.viewAllText will still work here, because in the universal results component we translate deprecated config into their updated names.